mesheryctl/pkg/utils/error.go

```diff
@@ -2,11 +2,14 @@ package utils
 
 import (
 	"fmt"
+	"strconv"
 
 	"github.com/layer5io/meshkit/errors"
 )
 
 var (
+	ErrFailRequestCode     = "1034"
+	ErrFailReqStatusCode   = "1035"
```

Comment:
```suggestion
	ErrFailRequestCode     = "not set"
	ErrFailReqStatusCode   = "not set"
```

workflows will take care of it and assign a code to it.